tests/js/spec/components/collapsible.spec.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import {mountWithTheme} from 'sentry-test/enzyme';
+
+import Button from 'app/components/button';
+import Collapsible from 'app/components/collapsible';
+
+const items = [1, 2, 3, 4, 5, 6, 7].map(i => <div key={i}>Item {i}</div>);
+
+describe('Collapsible', function () {
+  it('collapses items', function () {
+    const wrapper = mountWithTheme(<Collapsible>{items}</Collapsible>);
+
+    expect(wrapper.find('div').length).toBe(5);
+    expect(wrapper.find('div').at(2).text()).toBe('Item 3');
+
+    expect(wrapper.find('Button[data-test-id="expand"]').text()).toBe(
+      'Show 2 collapsed items'
+    );
```

Comment:
imo we shouldn't use `data-test-id` for Buttons, we should instead use `aria-label` as that'll build good habits to ensure that our buttons are accessible.

Reply:
sure thing, let me switch to that